### DISCUSSION TITLE
Reset Password Emails

### DISCUSSION THREAD

**Discussion by Greenlight88** [cortexrd/Knack-Toolkit-Library#375] [Ideas]
I reached out to knack about this - there is no way to send out your own branded reset password link - this is what you get with the automated system

![image](https://github.com/user-attachments/assets/56a8c9c3-3653-48a9-b59b-a58107cfa53f)

(Note I edited this link, its just for show)

Not branded, low quality = lower trust 

If it were at all possible it would be great if we could firstly stop that email going out and secondly fire some sort of trigger to send our own email formatted how we like

**Comment by cortexrd** [comment]
Hi @Greenlight88,

I tried very hard, but it's not possible to change this mechanism.  The email is sent directly from Knack's server and the big gibberish string you see there is an encrypted token that prevents unauthorized users to reset the PW on your behalf.

There's no way to intercept this email and modify its HTML - and believe me... it's just perfect like that!

I wouldn't worry at all about the style and trust level.  Most password reset pages I've seen are all very basic and unbranded anyways.

Norm

**Reply by Greenlight88** [reply]
Hi Norm - thanks for trying You 're right I'm  overthinking it
